src/app/layout.js

```diff
@@ -1,5 +1,6 @@
 import localFont from "next/font/local";
 import "./globals.css";
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
PRAISE: Great job figuring out what file we're supposed to import bootstrap into!